neco-server-exporter/go.mod

```diff
@@ -0,0 +1,120 @@
+module github.com/cybozu/neco-containers/neco-server-exporter
+
+go 1.24.5
```

Comment:
nit: 1.24.5 has some vulnerabilities. I think the latest would be good.

Reply:
The version should exactly match with the workflow:
https://github.com/cybozu/neco-containers/blob/main/.github/workflows/main.yaml#L281
https://github.com/cybozu/neco-containers/actions/runs/19063539929/job/54448614430

Do you think it's ok to postpone to regular update, or better to update all for now?
```
$ git grep -F 1.24.5
.github/workflows/main.yaml:285:      go-version: "1.24.5"
admission/go.mod:3:go 1.24.5
bmc-log-collector/go.mod:3:go 1.24.5
cep-checker/go.mod:3:go 1.24.5
envoy/go.mod:3:go 1.24.5
neco-server-exporter/go.mod:3:go 1.24.5
squid-exporter/go.mod:3:go 1.24.5
tcp-keepalive/go.mod:3:go 1.24.5
```

Reply:
For now, I think it's enough to just update this one.

Reply:
> It's impossible to fix neco-server-exporter only, due to repository structure.

Do you mean we also have to update `.github/workflows/main.yaml:285`?
I think the others don’t need to be updated, right?

Reply:
> Theoretically yes with current CI implementation, but it will fail other containers' build on the next time.

Why do you think the other containers will fail to build?
My understanding is that even if we raise the go-version in the workflow(i.e. raise the Go version on the host), programs that specify 1.24.5 in their `go.mod` can be built.